features/machine/autoscaler.feature

```diff
@@ -52,3 +52,104 @@ Feature: Cluster Autoscaler Tests
     the expression should be true> machine_set.desired_replicas(cached: false) == 1
     """
     Then the machineset should have expected number of running machines
+  
+  # @author zhsun@redhat.com
+  # @case_id OCP-21516
+  @admin
+  @destructive
+  Scenario: Cao listens and deploys cluster-autoscaler based on ClusterAutoscaler resource
+    Given I have an IPI deployment
+    And I switch to cluster admin pseudo user
+
+    When I run the :create admin command with:
+      | f | https://raw.githubusercontent.com/openshift-qe/v3-testfiles/master/cloud/cluster-autoscaler.yml   |
+    Then the step should succeed
+    And admin ensures "default" clusterautoscaler is deleted after scenario
+    And 1 pods become ready with labels:
+      | cluster-autoscaler=default,k8s-app=cluster-autoscaler                                                 |
+
+  # @author zhsun@redhat.com
+  # @case_id OCP-21517
+  @admin
+  @destructive
+  Scenario: CAO listens and annotations machineSets based on MachineAutoscaler resource
+    Given I have an IPI deployment
+    And I switch to cluster admin pseudo user
+    Given I use the "openshift-machine-api" project
+    And I pick a random machineset to scale
+
+    When I run oc create over "https://raw.githubusercontent.com/openshift-qe/v3-testfiles/master/cloud/machine-autoscaler.yml" replacing paths:
```

Comment:
You didn't create the clusterautoscaler before creating the machineautoscaler, is the scenario valid?